include/RAJA/pattern/params/forall.hpp

```diff
@@ -43,45 +43,45 @@ struct ForallParamPack
 private:
   // Init
   template<typename EXEC_POL, camp::idx_t... Seq, typename... Args>
-  static constexpr void detail_init(EXEC_POL,
+  static constexpr void params_init(EXEC_POL const& pol,
```

Comment:
What are we planning on doing with this execution policy object?

Reply:
The plan is to pass it to the implementation functions as a normal argument instead of a template argument. This makes the call to the implementation normal overloading without any explicit template parameters. That is helpful as until c++20 you have to be more explicit about which functions you might be calling when explicitly passing template parameters, hence the detail:: that was at the call site of the implementations.

Reply:
That makes sense, and does clean up the call sites quite a bit.